Tasks/IISWebAppManagementOnMachineGroup/task.json

```diff
@@ -76,6 +76,14 @@
         }
     ],
     "inputs": [
+        {
+            "name": "EnableIIS",
+            "type": "boolean",
+            "label": "Enable IIS",
+            "required": false,
+            "defaultValue": "false",
+            "helpMarkDown": "Select the option to install IIS."
```

Comment:
Please put a more verbose help text.

Reply:
Please suggest some.

Reply:
"Check this if you want to install IIS on the machine"